test/helix.proj

```diff
@@ -0,0 +1,48 @@
+<Project DefaultTargets="Test">
+
+  <!-- Version included until we get global.json generation to support this SDK. -->
+  <Sdk Name="Microsoft.DotNet.Helix.Sdk" Version="1.0.0-beta.19052.1" />
+
+  <Target Name="Gather" BeforeTargets="Test">
+    <ItemGroup>
+      <ProjectsToTest Include="$(ProjectsToTest)" />
+    </ItemGroup>
+    <MSBuild Projects="@(ProjectsToTest)"
+              Targets="CreateTestPayload"
+              BuildInParallel="true">
+      <Output TaskParameter="TargetOutputs" ItemName="TestPayload" />
+    </MSBuild>
+    <ItemGroup>
+      <HelixWorkItem Include="%(TestPayload.TestName)">
+        <Command>%(TestPayload.Command)</Command>
+        <PayloadDirectory>%(TestPayload.Identity)</PayloadDirectory>
+        <Timeout>%(TestPayload.TestTimeout)</Timeout>
+      </HelixWorkItem>
+    </ItemGroup>
+  </Target>
+
+  <PropertyGroup>
+    <!-- TODO: Follow up with helix SDK -->
+    <SkipInvalidConfigurations>true</SkipInvalidConfigurations>
+    <HelixSource>pr/aspnet/extensions</HelixSource>
+    <HelixType>ci</HelixType>
+    <HelixBuild>private-$(USERNAME)</HelixBuild>
+    <HelixBuild Condition=" '$(CI)' == 'true' ">$(BUILD_BUILDNUMBER)</HelixBuild>
+    <IsExternal>true</IsExternal>
```

Comment:
Should this be false when building official builds internally?

Reply:
I'm not sure, I think we have to target different queues if we want this flag to be false (the jobs started failing when using the bot token for the current target queues, probably because we are using the .Open queues?)

Reply:
Ok. Can you add this detail in a comment for now? I don't think we have a reason to use anything but .Open queues for tests.